bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/accessor/DateTimeValueAccessor.java

```diff
@@ -37,7 +43,7 @@ protected void updateControlFromValue(Control control, Value value) {
 			setText(control, "");
 		} else {
 			Instant date = value.getInstantValue();
-			LocalDateTime localDateTime = LocalDateTime.ofInstant(date, ZoneId.of("UTC"));
+			LocalDateTime localDateTime = LocalDateTime.ofInstant(date, ZoneId.of(timezone));
```

Comment:
Wieso hier nicht auch die Methode aus DateTimeUtil verwenden?

Die Werte sollten injected werden:
```
String dateUtil = (String) InstancePreferenceAccessor.getValue(preferences, ApplicationPreferences.DATE_UTIL, DisplayType.DATE_UTIL, "", locale);
String timeUtil = (String) InstancePreferenceAccessor.getValue(preferences, ApplicationPreferences.TIME_UTIL, DisplayType.TIME_UTIL, "", locale);
```

Das `Locale locale` am besten aus dem CustomLocale, außer du bist sicher, dass `control.getData(TRANSLATE_LOCALE)` bei einer Änderung entsprechend geupdated wird.

Reply:
Die Methoden aus DateTimeUtil bringen nichts, da ich keine String Eingabe habe

Reply:
Aber an dieser Stelle geht es doch darum, einen String aus einem Instant zu erstellen (dieser wird dann in Zeilen 49-57 in das Control geschrieben). 
Ich dachte genau dafür ist die Methode DateTimeUtil#getDateTimeString gedacht.